source/manuals/programming-languages/nodejs/index.html.md.erb

```diff
@@ -445,21 +438,20 @@ for (let i = 0; i < 10; i += 1) {
 }
 
 // Over:
-for (let i=0,j=0;i<10 && j<10;j++,i=(j==10)?i+1:i,j=(j==10)?j=0:j,console.log(i,j)){}
+for (let i = 0, j = 0; i < 10 && j < 10; j++, i = (j == 10) ? i + 1 : i, j = (j == 10) ? j = 0 : j, console.log(i, j)) {}
```

Comment:
At a glance it seems a bit odd to add spacing to an example of code written to take up less space. Do you think it's needed here?

Reply:
Good point - I've removed this change.